app/api/ada/adaWallet.js

```diff
@@ -122,6 +122,8 @@ export const isValidMnemonic = (
   isValidEnglishAdaMnemonic(phrase, numberOfWords)
 );
 
+/** Wrapper function to create new Trezor ADA hardware wallet object 
+  * usage SINGLE public master key */
```

Comment:
I would change the wording of this comment. Maybe we could try with:

`it takes as an input the master public key` or something like that

Reply:
to make it simpler removed the second line of comment,
if we add only `it takes as an input the master public key` then it would be incomplete

Reply:
DONE